core/src/kvs/stash.rs

```diff
@@ -0,0 +1,17 @@
+use crate::idg::u32::U32;
+use crate::kvs::kv::Key;
+use std::collections::HashMap;
+
+#[derive(Default)]
+pub(super) struct Stash(pub HashMap<Key, U32>);
```

Comment:
I am wondering if we should not use a PrefixTree here? Not sure how big this collection can be

Reply:
To be honest, this collection will not be very big (in my understanding). I actually want to change this logic completely. It's now only used for Namespace IDs, Database IDs, and Table IDs which are generated within DEFINE statements.

I want to do this in a separate PR 👍.